README.md

```diff
@@ -103,8 +103,15 @@ $ python manage.py startapp myapp --template https://github.com/mongodb-labs/dja
 
 ### Configuring the `DATABASES` setting
 
-After you've set up a project, configure Django's `DATABASES` setting similar
-to this:
+After you've set up a project, configure Django's `DATABASES` with
```

Comment:
Are you going to update the project template? This also needs `import django_mongodb`. Should I use a URI even with a localhost configuration?

Reply:
Yes I'll update the project template and add the import statement to the README, and yes to localhost, here's how I'm running the tests:

```
DATABASE_URL = os.environ.get("DATABASE_URL", "mongodb://localhost:27017/djangotests")
```

Reply:
In a similar spirit to my comment below, I think promoting a URL as the default (even for something like localhost) is sending people in a more complicated direction in the long run, especially for Django developers coming to MongoDB. It seems pretty simple to parse a URI to the DATABASES format just by looking at it, or as I suggested, use `print(parse_uri(...))` if in doubt.

Reply:
Agree, I put it back the way it was, which fits the pattern of Django-way, followed by Mongo-way. @Jibola can you roll with this?

Reply:
Looks like we don't need `DISABLE_SERVER_SIDE_CURSORS` because it is PG specific

Reply:
**TL;DR**: I concede to this and in terms of how we dictate instructions for the a quickstart or target audience, I think that can be broken out of this one. 

The two conventional ways of getting a local MongoDB URI are:
* through [local atlas](https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-local/)
* through [`mongo-orchestration`](https://github.com/10gen/mongo-orchestration)
 but both of these methods -- in my opinion -- are still roads less traveled. 
 
I really do see the _average_ case being someone with an Atlas URI pointing to instructions on how to get an Atlas URI via the [Getting Started](https://www.mongodb.com/docs/atlas/getting-started/) page.

Because that average case ends up being a more involved cluster, I perceive the template shooting for those cases where no parsing or local db setup is done as the quickest way to get started. More advanced users in the Django and dev space will definitely be able to understand the parsing but true beginners to MongoDB I foresee still being confused.  

Nonetheless, ruminating over this has less to do with the implementation of `parse` and more to do with what gets spit out from our template. Since our template currently supports the `DATABASES` standard, we'll just keep the PR in alignment that way.